packages/graph/hash_graph/lib/graph/src/store/postgres/knowledge/entity/read.rs

```diff
@@ -41,16 +42,14 @@ impl<C: AsClient> crud::Read<Entity> for PostgresStore<C> {
         let mut compiler = SelectCompiler::new();
 
         let owned_by_id_index = compiler.add_selection_path(&EntityQueryPath::OwnedById);
-        let entity_uuid_index = compiler.add_distinct_selection_with_ordering(
-            &EntityQueryPath::Uuid,
-            Distinctness::Distinct,
-            None,
-        );
-        let version_index = compiler.add_distinct_selection_with_ordering(
-            &EntityQueryPath::Version,
+        let entity_uuid_index = compiler.add_selection_path(&EntityQueryPath::Uuid);
+        let record_id_index = compiler.add_distinct_selection_with_ordering(
```

Comment:
So happy that we aren't relying on hardcoding the indexes when adding new selections here